exporter/importables.go

```diff
@@ -2160,16 +2162,96 @@ var resourcesMap map[string]importable = map[string]importable{
+							if uc3LevelIdRegex.MatchString(se.EntityName) {
+								ic.Emit(&resource{
+									Resource: "databricks_registered_model",
+									ID:       se.EntityName,
+								})
+							}
+							// TODO: add else branch to emit databricks_model when we have support for it
+						}
+						// TODO: add else branch to emit UC function when we add support for them...
+					}
+					if se.InstanceProfileArn != "" {
+						ic.Emit(&resource{
+							Resource: "databricks_instance_profile",
+							ID:       se.InstanceProfileArn,
+						})
+					}
+					ic.emitSecretsFromSecretsPathMap(se.EnvironmentVars)
+					if se.ExternalModel != nil {
+						if se.ExternalModel.DatabricksModelServingConfig != nil {
+							ic.emitSecretsFromSecretPathString(se.ExternalModel.DatabricksModelServingConfig.DatabricksApiToken)
+						}
+						if se.ExternalModel.Ai21labsConfig != nil {
+							ic.emitSecretsFromSecretPathString(se.ExternalModel.Ai21labsConfig.Ai21labsApiKey)
+						}
+						if se.ExternalModel.AnthropicConfig != nil {
+							ic.emitSecretsFromSecretPathString(se.ExternalModel.AnthropicConfig.AnthropicApiKey)
+						}
+						if se.ExternalModel.AmazonBedrockConfig != nil {
+							ic.emitSecretsFromSecretPathString(se.ExternalModel.AmazonBedrockConfig.AwsAccessKeyId)
+							ic.emitSecretsFromSecretPathString(se.ExternalModel.AmazonBedrockConfig.AwsSecretAccessKey)
+						}
+						if se.ExternalModel.CohereConfig != nil {
+							ic.emitSecretsFromSecretPathString(se.ExternalModel.CohereConfig.CohereApiKey)
+						}
+						if se.ExternalModel.OpenaiConfig != nil {
+							ic.emitSecretsFromSecretPathString(se.ExternalModel.OpenaiConfig.OpenaiApiKey)
+						}
+						if se.ExternalModel.PalmConfig != nil {
+							ic.emitSecretsFromSecretPathString(se.ExternalModel.PalmConfig.PalmApiKey)
+						}
+					}
+				}
+			}
+			if mse.Config.AutoCaptureConfig != nil && mse.Config.AutoCaptureConfig.CatalogName != "" &&
+				mse.Config.AutoCaptureConfig.SchemaName != "" {
+				ic.Emit(&resource{
+					Resource: "databricks_schema",
+					ID:       mse.Config.AutoCaptureConfig.CatalogName + "." + mse.Config.AutoCaptureConfig.SchemaName,
+				})
+			}
 			return nil
 		},
 		ShouldOmitField: func(ic *importContext, pathString string, as *schema.Schema, d *schema.ResourceData) bool {
-			if pathString == "config.0.traffic_config" ||
-				(strings.HasPrefix(pathString, "config.0.served_models.") &&
-					strings.HasSuffix(pathString, ".scale_to_zero_enabled")) {
+			if pathString == "config.0.traffic_config" || pathString == "config.0.auto_capture_config.0.enabled" ||
+				(pathString == "config.0.auto_capture_config.0.table_name_prefix" && d.Get(pathString).(string) != "") {
 				return false
 			}
+			if res := servedEntityFieldExtractionRegex.FindStringSubmatch(pathString); res != nil {
+				field := res[2]
+				log.Printf("[DEBUG] ShouldOmitField: extracted field from %s: '%s'", pathString, field)
+				switch field {
```

Comment:
should we have a default case here?

Reply:
Default case could be also complex, so it's better to pass through and allow the last line of the function (call to the `defaultShouldOmitFieldFunc`) handle it